moveit_planners/chomp/chomp_motion_planner/src/chomp_trajectory.cpp

```diff
@@ -252,4 +253,17 @@ void ChompTrajectory::fillInMinJerk()
   }
 }
 
+void ChompTrajectory::fillInSeed(const moveit_msgs::MotionPlanRequest& req)
```

Comment:
I'd prefer that this function checked the request message for valid trajectory constraint data and return the corresponding boolean value. This would make it easier to use the ChompTrajectory class outside of the Chomp planner the as the calling code doesn't have to validate the request message; for instance one could write:
```
if(!trajectory.fillInSeed(req) )
{
  // failed to initialize trajectory from plan request so use a default initialization scheme
  trajectory.fillInLinearInterpolation();
}
```